src/ApplicationInsights/ApplicationInsights.Autorest/docs/Get-AzApplicationInsightsLinkedStorageAccount.md

```diff
@@ -0,0 +1,152 @@
+---
+external help file:
+Module Name: Az.ApplicationInsights
+online version: https://docs.microsoft.com/powershell/module/az.applicationinsights/get-azapplicationinsightslinkedstorageaccount
+schema: 2.0.0
+---
+
+# Get-AzApplicationInsightsLinkedStorageAccount
+
+## SYNOPSIS
+Returns the current linked storage settings for an Application Insights component.
+
+## SYNTAX
+
+### Get (Default)
+```
+Get-AzApplicationInsightsLinkedStorageAccount -ComponentName <String> -ResourceGroupName <String>
+ [-SubscriptionId <String[]>] [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+### GetViaIdentity
+```
+Get-AzApplicationInsightsLinkedStorageAccount -InputObject <IApplicationInsightsIdentity>
+ [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Returns the current linked storage settings for an Application Insights component.
+
+## EXAMPLES
+
+### Example 1: Get linked storage account associated with component "componentName"
+```powershell
+PS C:\> Get-AzApplicationInsightsLinkedStorageAccount -ResourceGroupName "rgName" -ComponentName "componentName"
```

Comment:
It will be better: `Get-AzApplicationInsightsLinkedStorageAccount -ResourceGroupName "rgName" -ComponentName "componentName"`